### DISCUSSION TITLE
Please Rename AuthorizeAttribute in Application layer.

### DISCUSSION THREAD

**Discussion by sechabamot** [jasontaylordev/CleanArchitecture#977] [Ideas]
The authorize attribute in the application layer that is meant for IRequest inheriting classes is really ambiguous as to Microsoft.AspNetCore.Authorization Authorize attribute. I am fairly new to this template and I can get how this is a small issue to most but I just spent hours trying to figure out why I couldn't get claims in my httpcontext.user property only to realize I was using the wrong attribute.

Nothing major. Just an idea so others don't fall into the same mistake.

**Comment by jasontaylordev** [comment]
Sorry about that @sechabamot, and thanks for opening this discussion. I think `[AuthorizeRequest]` could work well, since it will only work with Mediator requests through the associated `AuthorizationBehaviour`. Thoughts? I'd be interested in any suggestions; naming things is hard.

**Reply by sechabamot** [reply]
Hi Jason. 
That's exactly what I have renamed it in my local project. I think it's perfect.

Lol, I think you got the naming thing just fine.
While I have you here, want to thank you for the project template. I really like it.
